Codeinwp/otter-blocks
# Version 2.3

- https://github.com/Codeinwp/otter-blocks/issues/1635
- https://github.com/Codeinwp/otter-blocks/issues/1652
- https://github.com/Codeinwp/otter-blocks/issues/1622
- https://github.com/Codeinwp/otter-blocks/issues/1656
- https://github.com/Codeinwp/otter-blocks/issues/1637
- https://github.com/Codeinwp/otter-blocks/issues/1563
- https://github.com/Codeinwp/otter-blocks/issues/1614
- https://github.com/Codeinwp/otter-blocks/issues/1579
- https://github.com/Codeinwp/otter-blocks/issues/1609
- https://github.com/Codeinwp/otter-blocks/issues/1559
- https://github.com/Codeinwp/otter-blocks/issues/1608
- https://github.com/Codeinwp/otter-blocks/issues/1463
- https://github.com/Codeinwp/otter-blocks/issues/1466
- https://github.com/Codeinwp/otter-blocks/issues/575
- https://github.com/Codeinwp/otter-blocks/issues/1616
- https://github.com/Codeinwp/otter-blocks/issues/1627

## Testing Instructions

The big moment is here. A lot of features are packed. As always, please make sure that nothing crash. Things should be stable.

Changelog:


```
- Add Custom Taxonomy Support to Dynamic Values
- Add More New Fields to Form Block
- Add Option to Store Form Block Emails to WordPress Dashboard
- Add Api Field in Stripe Block
- Add Support for HTML in Stripe Block Messages
- Add CDN Links in Patterns
- Various Form Block Enhancements
- Bump Minimum PHP Compatibility to 5.6
- Fix Dynamic Tags Not Working in Widgets
- Fix Review Comparison Table Not Taking Reusable Review Blocks into Consideration
- Fix Unused Assets Being Loaded When a Block Is Removed from Widgets
- Fix Custom CSS Module Not Working with FSE
- Fix Accordion Block Schema Conflicts with Neve PRO Performance Module and Lazy-loading Off-screen Elements
```
